edge-hub/test/Microsoft.Azure.Devices.Routing.Core.Test/MessageTest.cs

```diff
@@ -72,15 +76,25 @@ public void TestSet()
                 Message3,
                 Message4,
                 Message6,
-                Message7
+                Message7,
+                Message9,
             };
 
-            Assert.Equal(4, messages.Count);
+            Assert.Equal(5, messages.Count);
             Assert.Contains(Message1, messages);
             Assert.Contains(Message3, messages);
             Assert.Contains(Message4, messages);
             Assert.Contains(Message6, messages);
+            Assert.Contains(Message9, messages);
             Assert.DoesNotContain(Message5, messages);
         }
+
+        [Fact]
+        [Unit]
+        public void MessageWithNullValueInPropertiesShouldNotThrow()
```

Comment:
Nit: can we rename this to something like MessageSizeWithNullPropertyValuesShouldNotThrow.. ?

Reply:
Sure, no problem.